akka-stream-tests/src/test/scala/akka/stream/scaladsl/FlowPrefixAndTailSpec.scala

```diff
@@ -90,6 +90,41 @@ class FlowPrefixAndTailSpec extends AkkaSpec {
       subscriber.expectSubscriptionAndComplete()
     }
 
+    "throw if tail is attempted to be materialized twice" in assertAllStagesStopped {
```

Comment:
these two tests fail sometimes because an OnSubsribe is missing. I have not yet figured out why.